### PR TITLE
fix: improve readme and fix log message #DA-360

### PR DESCRIPTION
**Pull request by ankur-arch**
- Fix misleading message of Prisma CLI is not installed instead of Failed to install Prisma CLI
- Clear project README and delegate to docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,171 +5,45 @@
...
-- [📖 &nbsp;Documentation](https://nuxt-prisma-five.vercel.app/)
-
 ## Features
 
 - Seamlessly set up Prisma CLI, Prisma schema, Prisma Migrate, and Prisma Client
 - Easily access Prisma Studio within Nuxt DevTools
 - Auto-imported `usePrismaClient()` composable for your Vue files
 
-## Quick setup
+### Quick setup
```
Comment (jharrell) [inline README.md:18]: I don't think this is correct. "Quick setup" isn't a child section of "Features". This should probably go back to being on the same level as "Features" and the following sections (You could make them all h3's though if you prefer.)

File: README.md [hunk 2]
```diff
@@ -5,171 +5,45 @@
...
-```
+   ```ts
+   export default defineNuxtConfig({
+     modules: ["@prisma/nuxt"],
+   });
+   ```
 
+3. Start the development server:
 
-<!-- Badges -->
```
Comment (jharrell) [inline README.md:164]: We need these for the badges at the top of the readme. Please don't delete them 🙏

File: README.md [hunk 3]
```diff
@@ -5,171 +5,45 @@
...
-  main()
-</script>
-```
+   ```ts
+   export default defineNuxtConfig({
+     modules: ["@prisma/nuxt"],
+   });
+   ```
 
+3. Start the development server:
```
Comment (ankur-arch) [inline README.md:35]:
```suggestion
<!-- Badges -->
```

File: README.md (hunk 2 above)
Comment (ankur-arch) [reply README.md:164]: Add that back in!

File: README.md [hunk 4]
```diff
@@ -1,19 +1,13 @@
-# Nuxt Prisma
+<!-- Badges -->
```
Comment (jharrell) [inline README.md:2]:
```suggestion
```

**Review by jharrell — APPROVED** [review]

**Pull request merged**